crates/vrm-spec/src/vrmc_vrm_1_0/mod.rs

```diff
@@ -120,7 +120,7 @@ pub struct MaterialColorBind {
     /// target color
     pub target_value: Vec<f64>,
 
-    pub material_color_bind_type: MaterialColorType,
+    pub r#type: MaterialColorType,
```

Comment:
memo: https://stackoverflow.com/questions/26611664/what-is-the-r-operator-in-rust

Reply:
spec schema: https://github.com/vrm-c/vrm-specification/blob/04ff34419bcd27eea548760dd88a7140ef16ce5f/specification/VRMC_vrm-1.0/schema/VRMC_vrm.expressions.expression.materialColorBind.schema.json